README.md

```diff
@@ -15,7 +15,7 @@ Currently, it mainly supports related work for AWS, especially [aws-account-oper
 #### Requirements
 
 - Go >= 1.19
-- make
+- make (gmake if on macOS)
```

Comment:
I'm recommending gmake on macOS, because it behaves differently from the built-in make and aligns with our Linux colleagues. For example, the built-in make doesn't seem to respect the PATH manipulation at the top of README.md.